shopfloor_stock_picking_partner_note/services/checkout.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 Camptocamp SA (https://www.camptocamp.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+import os
+
+from odoo.addons.shopfloor.services.checkout import Checkout
+
+
+class CheckoutExt(Checkout):
+    _inherit = "base.shopfloor.process"
+
+    def _data_for_packing_info(self, picking):
+        res = super()._data_for_packing_info(picking)
+        note = picking.note
+        if note:
+            res += "{}{}".format(os.linesep, note).strip()
```

Comment:
Why don't you put that in shopfloor module ? The `note` field is provided by `stock` module, you don't need a new dependency.

I guess you need to do the same in the cluster picking & zone picking scenario (cos-4214)

Reply:
Because we're using the added functionality implemented in the stock_picking_partner_note` module right?
`https://github.com/OCA/stock-logistics-workflow/pull/1504/files#diff-9ec7eee25885dbe0455eef41843ed79f5fbfd8c8579b90c09f05754ee93fef2cR11` 

Regarding the two other scenarios, this method is not used and I don't see any packing-related info already implemented there, that's why I didn't touch it.
If we want it there too, I think that's a different topic and could be implemented at a later stage.

Reply:
One module is filling the note, shopfloor is displaying the note. But they don't depend on each other. You could fill the note differently.